automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookDataSyncingSpec.scala

```diff
@@ -59,13 +61,16 @@ class NotebookDataSyncingSpec extends ClusterFixtureSpec with NotebookTestUtils
                 remoteContentSize shouldBe localContentSize
               }
 
+              val hashedUser = Some(String.format("%064x", new BigInteger(1, MessageDigest.getInstance("SHA-256").digest((gcsPath.bucketName + ":" + clusterFixture.cluster.creator).getBytes("UTF-8")))))
+
               eventually(timeout(Span(4, Minutes)), interval(Span(30, Seconds))) {
                 val gcsLockedBy: Option[String] = getLockedBy(gcsPath.bucketName, GcsBlobName(gcsPath.objectName.value)).unsafeRunSync()
                 val welderLockedBy: Option[String] = Welder.getMetadata(clusterFixture.cluster, gcsPath, isEditMode).lastLockedBy
 
                 gcsLockedBy should not be None
                 welderLockedBy should not be None
                 gcsLockedBy shouldBe welderLockedBy
+                gcsLockedBy shouldBe hashedUser
```

Comment:
what is `welderLockedBy` in the line before this?

Reply:
After welder updates the `lockedBy` metadata, we're reading the value from the bucket twice - once through welder (`welderLockedBy`) and once directly through gcs (`gcsLockedBy`). This basically checks to make sure that welder can correctly read the `lockedBy` metadata of a notebook, but we weren't checking to see if welder is correctly locking it in the first place, which is what the new check does.